app/views/catalog/_arclight_index_default.html.erb

```diff
@@ -0,0 +1,48 @@
+<div class="al-search-result-index-article row d-flex align-items-start">
+  <% counter = document_counter_with_offset(document_counter) %>
+  <div class="col-auto">
+    <%= render partial: 'arclight_document_header_icon', locals: { document: document }  %>
+  </div>
+
+  <div class="col col-no-left-padding d-flex flex-wrap">
+    <div class="documentHeader my-w-75 w-md-100 order-0" data-document-id="<%= document.id %>">
+      <h3 class="index_title document-title-heading">
+        <%= link_to_document document, document_show_link_field(document), counter: counter %>
+
+        <% document.extents.each do |extent| %>
```

Comment:
suggestion (possibly blocking): I wonder if we should propose this change upstream to avoid having to override the entire index_default partial. Do you think that there would be support for displaying multiple extents? The field is `ssm` after all.